### PR TITLE
Fix integration test failures

### PR DESCRIPTION
**Pull request by malakaganga**
## Purpose
Fix integration test failures

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1544,7 +1563,7 @@
     <properties>
         <!-- Carbon platform version comes here-->
 
-        <carbon.kernel.version>4.4.14</carbon.kernel.version>
+        <carbon.kernel.version>4.4.37-SNAPSHOT</carbon.kernel.version>
```
Comment (daneshk) [inline pom.xml]: Shall we upgrade to kernel released version

File: pom.xml [hunk 2]
```diff
@@ -1650,15 +1668,15 @@
...
+        <orbit.version.commons.httpclient>3.1.0.wso2v6</orbit.version.commons.httpclient>
         <commons-dbcp.version>1.2.2</commons-dbcp.version>
         <commons-logging.version>1.1</commons-logging.version>
         <commons-lang.wso2.version>2.6.0.wso2v1</commons-lang.wso2.version>
         <jackson.version>1.9.2</jackson.version>
         <amber.version>0.22.1358727-wso2v2</amber.version>
         
         <!-- Carbon Commons -->        
-        <carbon.commons.version>4.5.4</carbon.commons.version>
+        <carbon.commons.version>4.6.51-SNAPSHOT</carbon.commons.version>
```
Comment (daneshk) [inline pom.xml]: Shall we upgrade to commons released version

File: pom.xml (hunk 1 above)
Comment (malakaganga) [reply pom.xml]: This branch (master_java11) is related to giving JDK 11 support for the products. So still the kernel hasn't been released with JDK 11 support so we have used this snapshot version

File: pom.xml (hunk 2 above)
Comment (malakaganga) [reply pom.xml]: This branch (master_java11) is related to giving JDK 11 support for the products. So still the commons hasn't been released with JDK 11 support so we have used this snapshot version

**Pull request merged**